src/applications/simple-forms/21-0966/config/helpers.js

```diff
@@ -115,115 +115,194 @@ export const initializeFormDataWithPreparerIdentification = preparerIdentificati
 };
 
 // Confirmation Page
-export const getClaimType = data => {
-  const benefitSelection = Object.keys(data.benefitSelection).filter(
-    key => data.benefitSelection[key],
+const benefitSelections = data =>
+  Object.keys(data.benefitSelection).filter(key => data.benefitSelection[key]);
+
+const alreadySubmittedBenefitIntents = alreadySubmittedIntents =>
+  Object.keys(alreadySubmittedIntents).filter(
+    key => alreadySubmittedIntents[key],
   );
 
-  if (
-    preparerIsSurvivingDependent({ formData: data }) ||
-    preparerIsThirdPartyToASurvivingDependent({ formData: data })
-  ) {
-    return 'pension claim for survivors';
-  }
-  if (
-    benefitSelection.includes(veteranBenefits.compensation) &&
-    benefitSelection.includes(veteranBenefits.pension)
-  ) {
-    return 'disability compensation and pension claims';
+const isAlreadySubmitted = (data, alreadySubmittedIntents) => {
+  const newlySelectedBenefit = benefitSelections(data).filter(
+    benefitSelection => {
+      if (alreadySubmittedBenefitIntents(alreadySubmittedIntents).length > 0) {
```

Comment:
hm i don't have a specific recommendation here, but i did find this a bit hard to read. no change necessary. I think the more return statements you have in general, the more confusing a function becomes.

Reply:
Thanks @rhasselle-oddball ! Does it look better now?